src/System.Linq.Parallel/tests/PlinqModesTests.cs

```diff
@@ -2,165 +2,136 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading;
+using System.Threading.Tasks;
 using Xunit;
 
 namespace Test
 {
     public static class PlinqModesTests
     {
-        private static IEnumerable<Labeled<Action<ParVerifier, ParallelQuery<int>>>> EasyUnorderedQueries()
+        private static IEnumerable<Labeled<Action<UsedTaskTracker, ParallelQuery<int>>>> EasyUnorderedQueries(int count)
         {
             // Some queries may be brittle, depending on source type - tests may fail to be run in parallel by default..
```

Comment:
Is this comment still valid?  It seems like we just always pass count into the Take method in the query.